### ISSUE TITLE
Domain and Project

### ISSUE THREAD

**Issue by ThetaEdra** [pavelkomarov/exportify#20]
It seems like this Project is way behind other Versions by now for example https://github.com/watsonbox/exportify who offers a lot of features and even export of the favourites playlist. Id suggest to redirect the projects web Page exportify.net to the for now more advanced https://watsonbox.github.io/exportify/ page. If id have knewn about the other fork/project id have used that one instead and i feel like the webpage should include a link to the latest and most advanced version. Especially since this Project seems rather abondoned.

**Comment by pavelkomarov** [comment]
The story is @watsonbox wrote the original, then didn't update it for years. I wrote this version so that I could do the analytics I wanted. Then he came back and updated his own version.

**Comment by ThetaEdra** [comment]
Oh i didnt know that.
Is there any plan to get in contact with the original author?

**Comment by pavelkomarov** [comment]
Yeah, we've talked. I opened a PR over on his version, and he looked through this version's code pretty thoroughly. Not sure what he used, other than some ideas.

**Comment by watsonbox** [comment]
Yes for reference the PR is [here](https://github.com/watsonbox/exportify/pull/70). I did indeed try to use what I could, though it wasn't straightforward because of the different approaches I took with React and rate limiting, and the removal of automated tests from this fork, for example.

I agree that the domain [exportify.net](https://exportify.net/) does rather cement this fork as the "canonical version", which at this point probably isn't optimal for users. As the owner of the domain, that is in @pavelkomarov's hands, and of course he's perfectly entitled to leave things as they are.
